src/components/edge/PotentialEdge.tsx

```diff
@@ -17,25 +17,30 @@ export interface PotentialEdgeProps {
 }
 
 const getCurve = (
-    src: Position, dest: Position, edgeStyle: EdgeStyle, className: string, showArrowhead?: boolean
-  ) =>
-    <EdgeCurve src={src} dest={dest} edgeStyle={edgeStyle} className={className} showArrowhead={showArrowhead} />;
-
-const PotentialEdge = (props: Preact.RenderableProps<PotentialEdgeProps>): JSX.Element => {
-  const { src, dest, className, edgeStyle, showArrowhead } = props;
+  src: Position,
+  dest: Position,
+  edgeStyle: EdgeStyle,
+  className: string,
+  showArrowhead?: boolean,
+) => <EdgeCurve src={src} dest={dest} edgeStyle={edgeStyle} className={className} showArrowhead={showArrowhead} />;
```

Comment:
I'm unsure of switching to the syntax of a list of params, can we just do

`({ src, dest, edgeStyle, className, showArrowHead }: PotentialEdgeProps)`

instead?

Reply:
Sure. Will do.